mod/community.php

```diff
@@ -35,6 +36,25 @@ function community_content(App $a, $update = 0)
 		return;
 	}
 
+	$accounttype = null;
+
+	if ($a->argc > 2) {
```

Comment:
Would it be better to use a special parameter in the query string to filter for content?
It would give us the possibility (in the future) to combine different filters.